server/system/service/auth_credentials.go

```diff
@@ -228,6 +229,15 @@ func (svc *auth) createUserToken(ctx context.Context, u *types.User, kind string
 
 			// random password string, "3i[g0|)z"
 			token = fmt.Sprintf("%s", rand.Password(credentialsTokenLength))
+		case credentialsTypeInviteEmailToken:
+			expSec := svc.settings.Auth.Internal.SendUserInviteEmail.Expires
+			if expSec == 0 {
+				expSec = 72
```

Comment:
Prefer to avoid magic numbers.
In this case, it's ok (should be trivial to figure out) but for more complex & used multiple times cases, define a (package/global) constant